providers/aws-s3/src/test/java/org/jclouds/aws/s3/filter/AwsRequestAuthorizeSignatureV4Test.java

```diff
@@ -0,0 +1,53 @@
+package org.jclouds.aws.s3.filter;
```

Comment:
Please add license header to resolve CI failure.

Reply:
added